include/vrv/functorparams.h

```diff
@@ -1332,6 +1332,30 @@ class ConvertToUnCastOffMensuralParams : public FunctorParams {
     ArrayOfObjects m_segmentsToDelete;
 };
 
+//----------------------------------------------------------------------------
+// CountClassIdsParams
+//----------------------------------------------------------------------------
+
+/**
+ * member 0: a map with classId/count pairs
+ * member 1: whitelist, a vector of classIds that should only be added
+ * member 2: blacklist, a vector of classIds that should be ignored
+ **/
+
+class CountClassIdsParams : public FunctorParams {
+public:
+    CountClassIdsParams()
+    {
+        m_elementClassIdCount.clear();
+        m_include.clear();
+        m_exlude.clear();
+    }
```

Comment:
These are redundant as the members have default constructors.  Suggest
CountClassIdsParams() = default;